src/Core/Prefab/Globe/BuilderEllipsoidTile.js

```diff
@@ -30,21 +39,24 @@ BuilderEllipsoidTile.prototype.Prepare = function Prepare(params) {
 
 // get center tile in cartesian 3D
 BuilderEllipsoidTile.prototype.Center = function Center(params) {
-    params.center = params.extent.center().as('EPSG:4978').xyz();
+    params.center = params.extent.center(this.tmp.coords[0])
+        .as('EPSG:4978', this.tmp.coords[1]).xyz();
     return params.center;
 };
 
 // get position 3D cartesian
 BuilderEllipsoidTile.prototype.VertexPosition = function VertexPosition(params) {
-    params.cartesianPosition = new C.EPSG_4326_Radians(
-        params.projected.longitudeRad,
-        params.projected.latitudeRad).as('EPSG:4978');
-    return params.cartesianPosition;
+    this.tmp.coords[0]._values[0] = params.projected.longitudeRad;
+    this.tmp.coords[0]._values[1] = params.projected.latitudeRad;
+
+    this.tmp.coords[0].as('EPSG:4978', this.tmp.coords[1]).xyz(this.tmp.position);
+    this.tmp.normal.copy(this.tmp.position).normalize();
```

Comment:
It would be necessary to fix this normal, because is normal to sphere and not ellipsoid.

I started in this commit https://github.com/iTowns/itowns/commit/75a5b87a2a9f907a08c557bf3ba5e48e9955efb5#diff-e74011929d1712ee6adcb01a09dd097aR51

The correct normal is computed in `Ellipsoid.cartographicToCartesian`.
You can change now or I do a new PR